ee/billing/quota_limiting.py

```diff
@@ -200,8 +200,15 @@ def update_all_org_billing_quotas(dry_run: bool = False) -> Dict[str, Dict[str,
     )
 
     teams: Sequence[Team] = list(
-        Team.objects.select_related("organization").exclude(
-            Q(organization__for_internal_metrics=True) | Q(is_demo=True)
+        Team.objects.select_related("organization")
```

Comment:
wow, this is surprising! In US:

(1) The posthog_team table is 51MB, so probably will never run into memory issues
(2) The posthog_organization table is 7GB 😳

Further analysis into (2) -> http://metabase-prod-us/question/477-posthog-org-size-analysis -> tells me the table columns themselves are still ~140MB, so it _shouldn't_ lead to OOMing unless we have some strict memory limit on celery tasks. But also interesting that indexes make up GBs of data for this table 😮 

Interested to see if this solves it. Default suggests no memory limit -> https://docs.celeryq.dev/en/stable/reference/cli.html#cmdoption-celery-worker-max-memory-per-child